packages/hooks/src/use-local-storage.tsx

```
@@ -13,8 +12,7 @@ export const getValueFromLocalStorage = (key: string, defaultValue: any) => {
 };
 
 export const setValueIntoLocalStorage = (key: string, value: any) => {
-  if (typeof window === undefined || typeof window === "undefined")
-    return false;
+  if (typeof window === undefined || typeof window === "undefined") return false;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)